src/runtime/Makefile

```diff
@@ -175,7 +176,8 @@ HYPERVISORS := $(HYPERVISOR_ACRN) $(HYPERVISOR_FC) $(HYPERVISOR_QEMU) $(HYPERVIS
 QEMUPATH := $(QEMUBINDIR)/$(QEMUCMD)
 QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
 
-QEMUTDXPATH := $(QEMUBINDIR)/$(QEMUTDXCMD)
+#QEMUTDXPATH := $(QEMUBINDIR)/$(QEMUTDXCMD)
```

Comment:
Are you leaving the originals here to make it easy to revert once we have upstream qemu supported? 

PS: I don't have an issue with this, just checking.

Reply:
That's exactly the idea. :-)